File: sdk/dotnet/lmsservice_db/src/SDK.cs

using lmsservice_db.src.api.course.module.course.moduleservice;
using lmsservice_db.src.api.course.content.course.contentservice;
using lmsservice_db.src.api.course.courseservice;
using lmsservice_db.src.api.learning.path.courses.learning.path.coursesservice;
using lmsservice_db.src.api.learning.path.learning.pathservice;
using lmsservice_db.src.api.user.learning.user.learningservice;
using lmsservice_db.src.api.certificates.certificatesservice;


namespace lmsservice_db.SDK;
public class SDK
{
    public APIClient Client { get; set; }
    public CourseModule CourseModule { get; set; }
public CourseContent CourseContent { get; set; }
public Course Course { get; set; }
public LearningPathCourses LearningPathCourses { get; set; }
public LearningPath LearningPath { get; set; }
public UserLearning UserLearning { get; set; }
public Certificates Certificates { get; set; }

    public SDK(string baseUrl)
    {
        Client = new APIClient(baseUrl);
        CourseModule = new CourseModule(Client);
CourseContent = new CourseContent(Client);
Course = new Course(Client);
LearningPathCourses = new LearningPathCourses(Client);
LearningPath = new LearningPath(Client);
UserLearning = new UserLearning(Client);
Certificates = new Certificates(Client);


    }
}

File: sdk/node/src/api/sdk.ts

import AuthenticationService from './authentication';
import { CourseModule } from '../api/course.module/course.module'
import { CourseContent } from '../api/course.content/course.content'
import { Course } from '../api/course/course'
import { LearningPathCourses } from '../api/learning.path.courses/learning.path.courses'
import { LearningPath } from '../api/learning.path/learning.path'
import { UserLearning } from '../api/user.learning/user.learning'
import { Certificates } from '../api/certificates/certificates'

import { APIClient } from './client';

export class SDK {
  public APIClient: APIClient;
  public auth: AuthenticationService;
public course_modules: CourseModule;
public course_contents: CourseContent;
public courses: Course;
public learning_path_courses: LearningPathCourses;
public learning_paths: LearningPath;
public user_learnings: UserLearning;
public certificates: Certificates;


  constructor() {
     this.APIClient = new APIClient();
     this.auth = new AuthenticationService(this.APIClient);
this.course_modules = new CourseModule(this.APIClient);
this.course_contents = new CourseContent(this.APIClient);
this.courses = new Course(this.APIClient);
this.learning_path_courses = new LearningPathCourses(this.APIClient);
this.learning_paths = new LearningPath(this.APIClient);
this.user_learnings = new UserLearning(this.APIClient);
this.certificates = new Certificates(this.APIClient);

  }
}